CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+### Added
+- CRUD operations for OSIDB entities Flaw Comment (create, retreive, list),
```

Comment:
is it intentional not having a delete for Flaw Comment ?

Reply:
Yes it is, as there is no option to delete comments in bugzilla, once they are created, they stay there.